examples/shouldi/shouldi/python/pypi.py

```diff
@@ -60,8 +52,6 @@ async def pypi_package_url(response_json: Dict["str", Any]) -> str:
 
 
 @op(
```

Comment:
We'll want to move this function above the `pypi_package_json`, so that it's generated input can be used by `pypi_package_json`.

```suggestion
@op(
    outputs={"directory": run_bandit.op.inputs["pkg"]},
```

Reply:
thanks, done